src/dracula.yml

```diff
@@ -107,106 +139,7 @@ tokenColors:
       foreground: *FG
   
   # ALL AVAILABLE UI SCOPES
-  # "tab.activeForeground": "",
```

Comment:
Are all the scopes that we aren't using nulled out above? If so, disregard.

If not, can we either add the scopes that we're not using above commented or nulled out or keep the non-used scopes commented out here?

I find it helpful to be able to compare what scopes are being used vs not being used.